OpenOversight/app/static/js/find_officer.js

```diff
@@ -1,89 +1,106 @@
-$(document).ready(function() {
-
-    var navListItems = $('ul.setup-panel li a'),
-        allWells = $('.setup-content');
+function buildSelect(name, data_url, dept_id) {
```

Comment:
It's a long term goal, but it'd be good to move away from jquery at some point if we could.  That being said, this looks great.